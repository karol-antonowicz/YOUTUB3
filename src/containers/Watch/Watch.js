import React from 'react';
import './Watch.scss';
import { Video } from '../../components/Video/Video'
import { RelatedVideos } from '../../components/RelatedVideos/RelatedVideos';
import { VideoMetaData } from '../../components/VideoMetaData/VideoMetaData';

export const Watch = props => {
    return (
        <div className='watch-grid'>
            <Video className='video' id='0beu6BodTYs' />
            <VideoMetaData className='metadata' viewCount={1520}/>
            <div className='metadata' style={{ width: '100%', height: '100px', background: '#F91112' }}>Metadata</div>
            <div className='video-info-box' style={{ width: '100%', height: '100px', background: '#BD10E0' }}>Video Info box</div>
            <div className='comments' style={{ width: '100%', height: '100px', background: '#9013FE' }}>comments</div>
            <RelatedVideos className='relatedVideos' />
        </div>
    )
}