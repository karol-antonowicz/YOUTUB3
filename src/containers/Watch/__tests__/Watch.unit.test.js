import React from 'react';
import {shallow} from 'enzyme';
import {Watch} from '../Watch';

describe('Watch testing', () => {
    test('Renders ok', ()=> {
        const wrapper = shallow(<Watch/>);
        expect(wrapper).toMatchSnapshot();
    });
});