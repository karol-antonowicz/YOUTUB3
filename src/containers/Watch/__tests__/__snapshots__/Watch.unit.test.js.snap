// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Watch testing Renders ok 1`] = `
<div
  className="watch-grid"
>
  <Video
    className="video"
    id="0beu6BodTYs"
  />
  <VideoMetaData
    viewCount={1520}
  />
  <VideoInfoBox />
  <div
    className="comments"
    style={
      Object {
        "background": "#9013FE",
        "height": "100px",
        "width": "100%",
      }
    }
  >
    comments
  </div>
  <RelatedVideos
    className="relatedVideos"
  />
</div>
`;
